keel-sql/src/test/resources/application.properties

```
@@ -11,6 +11,6 @@ spinnaker.baseUrl=http://spinnaker
 services.fiat.baseUrl=https://fiat.net
 spring.application.name=keel
 sql.enabled=true
-sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename
-sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename
+sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/keel&TC_REUSABLE=true
+sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/keel&TC_REUSABLE=true
```

tiny cleanup (as per https://www.testcontainers.org/modules/databases/jdbc/):
```suggestion
sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22:///keel&TC_REUSABLE=true
sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22:///keel&TC_REUSABLE=true
```

Since host & port are assigned dynamically, TC's JDBC proxy driver ignores passed values, and JDBC URL allows providing empty values, hence `:///` (`://` + emptyness + `/`)